ntbs-service/Models/Entities/MBovisExposureToKnownCase.cs

```diff
@@ -27,11 +26,15 @@ public class MBovisExposureToKnownCase : ModelBase, IHasRootEntityForAuditing
         [Display(Name = "Exposure setting")]
         public ExposureSetting? ExposureSetting { get; set; }
         
-        [Required]
+        [RequiredIf(@"NotifiedToPheStatus == Enums.Status.Yes", ErrorMessage = "This field is required")]
```

Comment:
I think we can do with a better error message here - at least "NTBS ID is required".
Let's put it in the file with all the other messages, too! If you have a look in there, there are a couple of generic messages akin to what you're trying to do, but which use the field name!